tools/i18n-toolkit/README.md

```diff
@@ -26,9 +26,33 @@ yarn add @gooddata/i18n-toolkit -D
 
 Toolkit is used from command line and has some options to enabled all checks. By default, all check are disabled and need to be turn on by one by for all cases.
 
-#### Option `-p, --path <type>`
+#### Option `-p, --path <path>`
 
-This option is **required** and it is path for directory where are translations jsons. There can be lots of `*.json` files, all files are behaving as a localisations files and need to be valid. There need to by file `en-US.json` that is used as a default file for other validations. Without this path and file, validator can not work.
+This option is **required** (if not `--paths` defined) and it is path for directory where are translations jsons. There can be lots of `*.json` files, all files are behaving as a localisations files and need to be valid. There need to be file `en-US.json` that is used as a default file for other validations. Without this path and file, validator can not work.
+
+> **Caveats**
+>
+> You can specify more paths by repeating `--path` argument and also you can use `--paths` argument instead.
+
+**Example**
+
+```
+ i18n-toolkit --path src/share/translations --path src/Component/translations
+```
+
+#### Option `-ps, --paths <paths>`
```

Comment:
Can we break this? I am not a fan of two very similar ways of specifying the same thing. Can we maybe just keep the `paths` variant and get rid gf the `path` one? Because the `paths` is strictly better than `path` so I would get rid of it while the package is not used by anyone.